src/mlpack/methods/linear_svm/linear_svm.hpp

```diff
@@ -94,14 +94,15 @@ class LinearSVM
    * @paran delta Margin of difference between correct class and other classes.
    * @param optimizer Desired optimizer.
    */
-  template <typename OptimizerType = ens::L_BFGS>
+  template <typename OptimizerType = ens::L_BFGS, typename... CallbackTypes>
   LinearSVM(const MatType& data,
             const arma::Row<size_t>& labels,
-            const size_t numClasses = 2,
-            const double lambda = 0.0001,
-            const double delta = 1.0,
-            const bool fitIntercept = false,
-            OptimizerType optimizer = OptimizerType());
+            const size_t numClasses,
+            const double lambda,
+            const double delta,
+            const bool fitIntercept,
+            OptimizerType optimizer,
+            CallbackTypes&&... callbacks);
```

Comment:
Can you please add description for callbacks in this constructor and now this is not default we should remove that part also.

Reply:
Done.